architecture/networking/routes.adoc

```diff
@@ -151,8 +151,10 @@ ciphers for the connection to be complete:
 See the link:https://wiki.mozilla.org/Security/Server_Side_TLS[Security/Server
 Side TLS] reference guide for more information.
 
-The router defaults to the `intermediate` profile. You can select a different
-profile using the `--ciphers` option when creating a router, or by changing
+By default the router selects the intermediate profile and sets ciphers based on this profile.
+When a profile is selected, only the ciphers are set. The TLS version is not governed by the profile.
```

Comment:
Do we provide information about setting the TLS version elsewhere?

Reply:
The link to the Mozilla page has all the version specific info.

In this context (I believe) you're configuring your router to handle connections from clients supporting a specific version of TLS.

So, for example, if you wanted to only support secure connections from clients using TLS 1.3, you'd set the router to the Modern profile, and anyone using a client implementing an older version of TLS would have their connection request rejected.

Reply:
*this link - https://wiki.mozilla.org/Security/Server_Side_TLS